pkgs/by-name/fi/firefox-sync-client/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "firefox-sync-client";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "Mikescher";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-CNTTlBYJlQ4z6VXgi/mZpwxIAVOxJUnIL9flyvEzmXg=";
+  };
+
+  vendorHash = "sha256-Gb+4fxMBgvPB9Ki7zIwscY9l2kJ+tuE1Mc3W08YTfk8=";
+
+  meta = with lib; {
+    description = "A commandline-utility to list/view/edit/delete entries in a firefox-sync account.";
+    homepage = "https://github.com/Mikescher/firefox-sync-client";
+    license = licenses.asl20;
+    maintainers = [ ];
```

Comment:
Why not add yourself as a maintainer? We should avoid adding orphaned packages if possible, especially if the package doesn't have an update script